views/bot_image.erb

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>EveryPolitician Rebuilder</title>
+    <style>
+      body {
+        max-width: 400px;
+        margin: 0 auto;
+        padding: 10px;
+        background-color: #fff;
+      }
+    </style>
+  </head>
+  <body>
+    <a href="https://medium.com/@everypolitician/getting-busy-with-scraper-data-957a2ddd9963">
+      <img alt="EveryPolitician Bot" src="https://avatars0.githubusercontent.com/u/12693984?v=3&amp;s=800" width="400" height="400">
```

Comment:
miniscule nit, and not worth changing, but for future reference `v=3;s=800` will work here (I'm not up on the arguments as to whether `;` or `&` is more correct, but I always prefer ';' if only just because you don't need to worry about encoding it like this…)